corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -68,6 +68,16 @@
             </a>
           {% endif %}
           {% include "users/partials/edit_commtrack_user_settings.html" %}
+          {% if support_info.locations %}
```

Comment:
Yeah, I would love to have this integrated into the main widgets.  Couldn't figure out how to get it as part of the form or widget since all we have there are the location IDs and a select2 endpoint (no names or types).  Should definitely be doable with a redesign though.

Reply:
It's a lot like the CRUD pages (API keys under My Account is a good example), but I think those are set up to control the whole page, not to work within a tab.